services/webdav/pkg/service/v0/service.go

```diff
@@ -346,7 +346,11 @@ func (g Webdav) Thumbnail(w http.ResponseWriter, r *http.Request) {
 		switch e.Code {
 		case http.StatusNotFound:
 			// StatusNotFound is expected for unsupported files
-			renderError(w, r, errNotFound(notFoundMsg(tr.Filename)))
+			message := notFoundMsg(tr.Filename)
+			if strings.Contains(e.Detail, "could not find space") {
+				message = spaceNotFoundMsg(tr.Filename, user.Username)
```

Comment:
We can't use `user.Username` as space name. It could be a project space.

Reply:
Maybe I should come up with suitable function name

Reply:
I see that the handler `Thumbnail` handles the `old/new` webdav paths only. Is there any chance that it will also used for project spaces?
https://github.com/owncloud/ocis/blob/4c5d58f53ba5e9ac66ee11f5ae7b28530a49a413/services/webdav/pkg/service/v0/service.go#L98-L118

Reply:
Yes it is also for project spaces. But let's think about this problem when we solved above discussion :wink: